contracts/DivviRegistry.sol

```diff
@@ -31,6 +39,21 @@ contract DivviRegistry is
     address indexed rewardsProvider,
     address indexed rewardsConsumer
   );
+  event ReferralRegistered(
+    address user,
+    address indexed rewardsProvider,
+    address indexed rewardsConsumer,
+    uint256 indexed chainId,
```

Comment:
I'm still wondering whether we should use our string network ID (`ethereum-mainnet`, `op-mainnet`, etc). 🤔 

It is more gas efficient to use the numeric version.
But wondering if we want to support non EVM chains.

Well, I see that protocols like hyperlane which support Solana [assigned it a chainId too](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/0e663ad85ddb681b684d80212039e8ae5c82b1c7/rust/main/hyperlane-core/src/chain.rs#L171C21-L171C31).
And I think Wormhole and LayerZero do something similar.